src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/VisitVisitor.kt

```diff
@@ -27,16 +27,15 @@ data class VisitVisitor(
   @Column(name = "OFFENDER_VISIT_VISITOR_ID", nullable = false)
   val id: Long? = null,
 
-  @ManyToOne(optional = false, fetch = FetchType.LAZY)
-  @JoinColumn(name = "OFFENDER_BOOK_ID", nullable = false)
-  val offenderBooking: OffenderBooking,
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "OFFENDER_BOOK_ID")
+  val offenderBooking: OffenderBooking? = null,
 
   @Column(name = "OFFENDER_VISIT_ID", nullable = false)
   val visitId: Long,
 
-  @ManyToOne(optional = false, fetch = FetchType.LAZY)
-  @JoinColumn(name = "PERSON_ID", nullable = false)
-  val person: Person,
+  @Column(nullable = false)
+  val personId: Long? = null,
 
```

Comment:
is this for the dummy row scenario with offender booking?

Reply:
No its for all - dummy or visitors. Ive not 'hidden' that or done anything clever like filters etc